spec/e2e/param_expansion/ops.yml

```diff
@@ -0,0 +1,38 @@
+hooks:
+  before:
+    - echo app before hook
+actions:
+  expansion:
+    command: |-
+      echo "First: ${1}";
+      echo "Second: ${2}";
+      echo "Third: ${3}";
+    param_expansion: true
+  numargs:
+    command: echo "$#"
+    param_expansion: true
+  shebang_bash:
+    command: |-
+      #!/usr/bin/env bash
+      echo "First: ${1}";
+      echo "Second: ${2}";
+      printf 'Other: %s\n' "${@:3}";
+      [ -t 0 ] || cat -;
+    param_expansion: true
+  shebang_python:
+    command: |-
+      #!/usr/bin/env python3
+      import sys, select
+      print(f"hello, +{'>> '.join(sys.argv[1:])}+")
+      if select.select([sys.stdin, ], [], [], 0.0)[0]:
+        print(sys.stdin.read().strip())
+    param_expansion: true
+  shebang_ruby:
+    command: |-
+      #!/usr/bin/env ruby
```

Comment:
This is pretty cool.

Reply:
Thanks! This was an idea I had early on in my use of `ops`. In fact, one of my initial tests of `ops` capabilities was seeing if it would honour a shebang in an action. I considered taking a stab at it pre-`2.0.0`, but was concerned it would introduce costs in performance. Though, with the advent of `crops`, and it's perf boosts, all concerns went out the window.